packages/lesswrong/lib/events/withNewEvents.ts

```diff
@@ -38,15 +39,15 @@ export const useNewEvents = () => {
       setEvents({ ...events, eventId: event });
     }
     
-    void createLWEvent({ variables: { data: event } });
+    backgroundTask(createLWEvent({ variables: { data: event } }));
     return eventId;
   }, [events, createLWEvent]);
   
   const closeEvent = useCallback((eventId: string, properties: any={}): string => {
     let event = events[eventId];
     let currentTime = new Date();
     
-    void createLWEvent({
+    backgroundTask(createLWEvent({
```

Comment:
I think these are both inside of hooks, so don't need the wrapper

Reply:
Because it's inside `lib/` rather than `components/`, it needs either the wrapper or else an `eslint-disable-next-line`. Imo it's better to favor `backgroundTask` over `eslint-disable` in cases like this (and make sure wrapping with `backgroundTask` is harmless when it's unnecessary).